tests/testTriggers.go

```diff
@@ -647,6 +649,70 @@ func TriggerRestartVolDriver(contexts *[]*scheduler.Context, recordChan *chan *E
 	})
 }
 
+// TriggerRestartKvdbVolDriver restarts volume driver where kvdb resides and validates app
+func TriggerRestartKvdbVolDriver(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer ginkgo.GinkgoRecover()
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: RestartKvdbVolDriver,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+	Step("get kvdb nodes bounce volume driver", func() {
```

Comment:
This event cant be independent with RestartManyVolDriver event. Need to make sure that RestartManyVolDriver event is not running while you are triggering this event